ui/src/styles/_layouts.scss

```diff
@@ -76,6 +82,9 @@
   &-left-largest {
     margin-left: var(--largestSpace);
   }
+  &-right {
```

Comment:
lets make sure the class matches the property, so like left-largetst below is largestSpace, since this is smallSpace it should be left-small